docs/static/fb-ls-kafka-example.asciidoc

```diff
@@ -0,0 +1,140 @@
+[[use-filebeat-modules-kafka]]
+=== Example: Set up {filebeat} modules to work with Kafka and {ls}
+
+This section shows how to set up {filebeat}
+{filebeat-ref}/filebeat-modules-overview.html[modules] to work with {ls} when
+you are using Kafka in between {filebeat} and {ls} in your publishing pipeline.
+The main goal of this example is to show how to load ingest pipelines from
+{filebeat} and use them with {ls}.
+
+The examples in this section show simple configurations with topic names hard
+coded. For a full list of configuration options, see documentation about
+configuring the <<plugins-inputs-kafka,Kafka input plugin>>. Also see
+{filebeat-ref}/kafka-output.html[Configure the Kafka output] in the _{filebeat}
+Reference_.
+
+==== Set up and run {filebeat}
+
+. If you haven't already set up the {filebeat} index template and sample {kib}
+dashboards, run the {filebeat} `setup` command to do that now: 
++
+[source,shell]
+----------------------------------------------------------------------
+filebeat -e setup
+----------------------------------------------------------------------
++
+The `-e` flag is optional and sends output to standard error instead of syslog.
++
+A connection to {es} and {kib} is required for this one-time setup
+step because {filebeat} needs to create the index template in {es} and
+load the sample dashboards into {kib}. 
```

Comment:
Should we add an example snippet about how this is configured in filebeat? Or a link to filebeat reference?

Reply:
In an early draft, I showed the command for disabling LS and enabling ES, but that looked weird since I tell the user to configure the output later on. I think a link is a good compromise, so I will do that.